web/src/App.jsx

```
@@ -5264,6 +5281,15 @@ function App() {
                       </div>
                     </div>
                     <div className="messages-list">
+                      {messages.length >= messageLimit && (
+                        <button
+                          className="ghost-btn"
+                          style={{ width: '100%', padding: '12px', fontSize: '0.9em' }}
+                          onClick={() => setMessageLimit(prev => prev + 50)}
+                        >
+                          Load older messages
+                        </button>
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)